getting-started/index.html

```diff
@@ -1,6 +1,13 @@
-<!doctype html>
-<meta charset="utf-8">
-<title>Redirect to home page</title>
-<script>window.location="https://www.nodetodo.org/";</script>
```

Comment:
With this `<script>` tag removed, won't it break the automatic redirect?

Reply:
I'll re-add fail of my part 😄